Resources/public/js/views/dashboard/ez-dashboardblockallcontentrowview.js

```diff
@@ -0,0 +1,189 @@
+        },
+
+        /**
+         * Tries to redirect a user to selected content edit
+         *
+         * @method _uiGoToContentEdit
+         * @protected
+         * @param event {Object} event facade
+         */
+        _uiGoToContentEdit: function (event) {
+            var content = this.get('content');
+
+            event.stopPropagation();
+
+            /**
+             * Makes request to redirect a user to content edit
+             *
+             * @event gotoContentEdit
+             * @param contentId {String} REST content id
+             * @param languageCode {String} content main language code
+             */
+            this.fire('gotoContentEdit', {
+                contentId: content.get('id'),
+                languageCode: content.get('mainLanguageCode')
+            });
+        },
+
+        /**
+         * Tries to redirect a user to selected content location preview
+         *
+         * @method _uiGoToContentPreview
+         * @protected
+         * @param event {Object} event facade
+         */
+        _uiGoToContentPreview: function (event) {
+            var content = this.get('content');
+
+            event.stopPropagation();
+
+            /**
+             * Makes request to redirect a user to content main location preview
+             *
+             * @event gotoContentPreview
+             * @param locationId {String} REST main location id
+             * @param languageCode {String} content main language code
+             */
+            this.fire('gotoContentPreview', {
+                locationId: content.get('resources').MainLocation,
+                languageCode: content.get('mainLanguageCode')
+            });
+        },
+
+        destructor: function () {
+            if (this._clickOutsideHandler) {
+                this._clickOutsideHandler.detach();
+            }
+        }
+    }, {
+        ATTRS: {
+            /**
+             * The content model
+             *
+             * @attribute content
+             * @type {Y.eZ.Content}
+             */
+            content: {},
+
+            /**
+             * The content type model
+             *
+             * @attribute contentType
+             * @type {Y.eZ.ContentType}
+             */
+            contentType: {},
+
+            /**
+             * The row options.
+             * Contains a list of button action view instances:
+             * - edit button,
+             * - preview button.
+             *
+             * @attribute options
+             * @type Array
+             */
+            options: {
+                valueFn: function () {
+                    return [
+                        new Y.eZ.ButtonActionView({
+                            actionId: 'edit',
+                            label: 'Edit'
+                        }),
+                        new Y.eZ.ButtonActionView({
+                            actionId: 'preview',
+                            label: 'Preview'
```

Comment:
Reminder, this is PlatformUI.
*Preview* has a meaning in PlatformUI but this is not what we'll be showed to the user when he'll use the corresponding link so as mentioned in the specification, use *View*.